indico/modules/rb_new/operations/bookings.py

```diff
@@ -162,13 +161,22 @@ def _bookings_query(filters):
         query = query.filter(db.or_(Reservation.booked_for_user == booked_for_user,
                                     Reservation.created_by_user == booked_for_user))
 
+    include_cancellations = filters.get('include_cancellations')
+    if include_cancellations:
+        query = query.filter(db.or_(ReservationOccurrence.is_cancelled,
+                                    ReservationOccurrence.is_rejected,
+                                    ReservationOccurrence.is_valid))
+    else:
+        query = query.filter(ReservationOccurrence.is_valid)
+
     return query
 
 
-def get_room_calendar(start_date, end_date, room_ids, **filters):
+def get_room_calendar(start_date, end_date, room_ids, include_cancellations=False, **filters):
     start_dt = datetime.combine(start_date, time(hour=0, minute=0))
     end_dt = datetime.combine(end_date, time(hour=23, minute=59))
-    query = _bookings_query(dict(filters, **{'start_date': start_dt, 'end_date': end_dt, 'room_ids': room_ids}))
+    query = _bookings_query(dict(filters, **{'start_date': start_dt, 'end_date': end_dt, 'room_ids': room_ids,
```

Comment:
why not just use kwargs here?

Reply:
also, let's rename the `<start|end>_date` filters here to `_dt` since that's what it actually uses (or move the combine logic into `_bookings_query`)